release/pkg/assets_capa.go

```diff
@@ -193,7 +193,7 @@ func (r *ReleaseConfig) GetAwsBundle(imageDigests map[string]string) (anywherev1
 		}
 	}
 
-	componentChecksum := generateComponentHash(artifactHashes)
+	componentChecksum := generateComponentHash(artifactHashes, r.DryRun)
```

Comment:
nit: we should maybe make `generateComponentHash` a `ReleaseConfig` receiver to avoid passing flags
Or maybe inject a `hashGenerator` on construction that would be different for dry run than for normal execution
We can do this separately

Reply:
Will make a note and implement this in a follow up